### ISSUE TITLE
docs have examples inconsistent with current version

### ISSUE THREAD

**Issue by osterbit** [qri-io/qri#471]
the example I came across was in `qri help add`

```
Examples:
  add a new dataset named annual_pop:
  $ qri add --data data.csv me/annual_pop
```
- (not sure if this is the best approach to dealing with but) would be cool if we extracted all the cobra examples into a table test

**Comment by dustmop** [comment]
This particular example was fixed by 62bf188edcf2c, but I agree that a better long-term, automatic solution would be preferable.

**Comment by ramfox** [comment]
closing because 1) the specific example was fixed and 2) while I agree it's cool to have an automatic solution, it's not on the roadmap as of now
